packages/core/src/video/Video.tsx

```diff
@@ -1,11 +1,32 @@
 import React from 'react';
-import {RemotionVideoProps} from './props';
+import {Sequence} from '../sequencing';
+import {RemotionMainVideoProps, RemotionVideoProps} from './props';
 import {VideoForDevelopment} from './VideoForDevelopment';
 import {VideoForRendering} from './VideoForRendering';
 
-export const Video: React.FC<RemotionVideoProps> = (props) => {
+export const Video: React.FC<RemotionVideoProps & RemotionMainVideoProps> = ({
+	startAt,
+	endAt,
+	...props
+}) => {
+	const startAtFrameNo = startAt ? startAt : 0;
+	const endAtFrameNO = endAt ? endAt : Infinity;
 	if (process.env.NODE_ENV === 'development') {
-		return <VideoForDevelopment {...props} />;
+		return (
+			<Sequence
+				from={0 - startAtFrameNo}
+				durationInFrames={endAtFrameNO - startAtFrameNo}
+			>
+				<VideoForDevelopment {...props} />;
+			</Sequence>
+		);
 	}
-	return <VideoForRendering {...props} />;
+	return (
+		<Sequence
+			from={0 - startAtFrameNo}
+			durationInFrames={endAtFrameNO - startAtFrameNo}
+		>
+			<VideoForRendering {...props} />;
+		</Sequence>
+	);
 };
```

Comment:
How about recursion? It would remove redudancy and be easier to reason about right?

```tsx
const {startAt, endAt, ...otherProps} = props
if (startAt || endAt) {
  return (
    <Sequence
      from={0 - startAtFrameNo}
	  durationInFrames={endAtFrameNO - startAtFrameNo}
    >
      <Video {...otherProps}>
    </Sequence>
  )
}
// Logic we had until now
```

Reply:
will surely do,